src/parse/errors.rs

```diff
@@ -682,19 +689,15 @@ impl Error {
         }
     }
 
-    pub(crate) fn unrecognized_subcommand(
-        subcmd: String,
-        name: String,
-        color: ColorChoice,
-    ) -> Self {
-        let mut c = Colorizer::new(true, color);
+    pub(crate) fn unrecognized_subcommand(app: &App, subcmd: String, name: String) -> Self {
+        let mut c = Colorizer::new(true, app.color());
 
         start_error(&mut c, " The subcommand '");
         c.warning(subcmd.clone());
         c.none("' wasn't recognized\n\n");
         c.warning("USAGE:");
-        c.none(format!("\n\t{} help <subcommands>...", name));
-        try_help(&mut c);
+        c.none(format!("\n\t{} <subcommands>...", name));
```

Comment:
```suggestion
        c.none(format!("\n    {} <subcommands>", name));
```

All other usage are 4 spaces. This needs to change the below test too.